File: challenges/3-area-trapezio/java/waaz/Trapezio.java
import java.util.Scanner;

public class Trapezio {

		public static double trapezio(double baseMaior,double baseMenor,double altura) {
			
			return (baseMaior + baseMenor) * (altura / 2);
			
		}
		public static void main(String[] args) {
			Scanner sc = new Scanner(System.in);
			double baseMaior = 0,baseMenor=0,altura=0;
			
			System.out.println("Base Maior: ");
			if(sc.hasNextDouble()) {	
				
				baseMaior = sc.nextDouble();
				
				System.out.println("Base menor");
				baseMenor = sc.nextDouble();
				
				System.out.println("Altura");
				altura = sc.nextDouble();
			}else {
				System.out.println("Valor invalido!. A aplicacoo so aceitara numeros. Tente novamente!! ");
			}		
			
			
			sc.close();
			
			System.out.println(trapezio(baseMaior,baseMenor,altura));
		}
}

File: challenges/3-area-trapezio/c++/feliper2002/trapezio.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

void areaTrapezio(float baseMaior, float baseMenor, float altura) {
    float area = ((baseMaior+baseMenor)*altura)/2;
    printf("A area do trapezio e: %.1f", area);
}

int main() {
    float baseMaior, baseMenor, altura;
    cout << "Base menor: "; cin >> baseMenor;
    cout << "Base maior: "; cin >> baseMaior;
    cout << "Altura: "; cin >> altura;
    areaTrapezio(baseMaior, baseMenor, altura);
    return 0;
}
